iacrcc/iacrcc.cls

```diff
@@ -161,44 +161,12 @@
   % doclicense added support for the macro \doclicenseLongTextForHyperref in version 3.1.0 (2022/04/27)
   \@ifpackagelater{doclicense}{2022/04/25}{\@doclicense@LongTextForHyperreftrue}{}
   \@ifpackageloaded{biblatex}{}{\bibliographystyle{iacrcc}}
-  % When pdflatex is used load TU fonts using fontenc
+  % When pdflatex is used load T1 fonts using fontenc
   \ifpdftex
-    \RequirePackage[TU]{fontenc}
+    \RequirePackage[T1]{fontenc}
```

Comment:
I guess my comment on the commit was lost here. I think we should use
```
\RequirePackage[TU,T1]{fontenc} to prefer unicode encoding over T1 encoding. There are only a few glyphs that require falling back to T1, since most of our fonts should now be unicode-encoded (e.g., latin modern).